### PR TITLE
Fixed UI issues of Dashboard, table and admin section

### PR DESCRIPTION
**Pull request by demon-36**
## Description

**Changed following files for proper working of their respective sections :** 

- Updated `admin.html` to fix the issue of not showing pending signup requests
- Updated `table.html` to fix the issue of not showing downloaded files 
- Updated `dashboard.html` to fix the issue of not showing pending file download requests
- Updated `DashCtrl.js` for proper working of `dashboard.html`

## Related Issue

Related Issue #755   and    [Related PR](https://github.com/scorelab/Bassa/pull/756)

## Motivation and Context

After submitting signup request and after starting downloading of added URL While switching through Admin and Completed section, Table for completed downloads and pending signup requests are missing and similarly for Dashboard section pending download requests are not shown.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
All of these changes are tested locally running Bassa server on localhost

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by demon-36** [comment]
@singhrohan62  @kmehant plz test Dashboard file changes on your local machine and also Did any of you also faced the `build.js` file error while running `gulp serve`

**Comment by demon-36** [comment]
@singhrohan62 I don't know what is causing this issue, actually It was fine with my machine :grin:

**Comment by singhrohan62** [comment]
Then I think we should wait for reviews from Nirmal and Milindu.

**Comment by vivonk** [comment]
> Well I checked the changes but found one disturbing. As you can see below, I have downloaded some files (images: 3, 6 and 4) and and added new one (image: 5)
> Screenshot from 2019-06-05 08-20-39
> Screenshot from 2019-06-05 08-20-42
> But when I started download for image 5, and when it was completed, the completed section didn't showed anything.
> Screenshot from 2019-06-05 08-21-07
> However, when I re-login then I can see all the images in the completed list. Check this on your own machine. See if this could be fixed. Else I have no objections for the merging 👍

@singhrohan62 this happens because the auth token expires in sometime so requests don't get any data. 

Better is if we can either increase the session expire time or once session expire redirect user to login page. 
@demon-36 what do you say?

**Comment by demon-36** [comment]
@vivonk I think it will be fine to redirect user to login page after session expires

File: ui/gulp/build.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ var $ = require('gulp-load-plugins')({
   pattern: ['gulp-*', 'main-bower-files', 'uglify-save-license', 'del']
 });
 
-const uglify = require('gulp-uglify-es').default;
+const uglify = require('gulp-uglify').default;
```
Comment (kmehant) [inline ui/gulp/build.js]: Can you please explain why this change is required as it breaks my gulp build script

Comment (kmehant) [reply ui/gulp/build.js]: The issue which you addressed seems to be an issue with your node-modules installed and not with the script.

Comment (demon-36) [reply ui/gulp/build.js]: Earlier I made a clean install of node_modules but I was facing the error while running `gulp serve` that's why I pushed this change, if this thing is only with my machine then it will be better to skip this change.
gulp-uglify

Comment (kmehant) [reply ui/gulp/build.js]: Please revert the change and also try installing that node module manually.

File: ui/src/app/dashboard/controllers/DashCtrl.js [hunk 2]
```diff
@@ -57,20 +57,17 @@
...
+      var data = response.data;
+      $scope.filterdownloads = _.filter(data, function(d) {return d.status==0});
+      $scope.downloads = _.map($scope.filterdownloads, function(obj) {
+           return _.extend({}, obj, {progress: 0});
       });
-    }
+    }, function(error){
+      ToastService.showToast("Oops! Something went wrong when fetching data");
+    });
+}
```
Comment (VibhorCodecianGupta) [inline ui/src/app/dashboard/controllers/DashCtrl.js]: This change is not necessary, the active downloads function is working just fine. The only problem was in the inline logical comparisons on the html pages. I've fixed those in my file management system PR.

Comment (VibhorCodecianGupta) [inline ui/src/app/dashboard/controllers/DashCtrl.js]: This change is not necessary, the active downloads function is working just fine. The only problem was in the inline logical comparisons on the html pages. I've fixed those in my file management system PR.

**Comment by kmehant** [comment]
@vivonk @agentmilindu @JaDogg bro we need this fix to be merged.

File: ui/gulp/build.js (hunk 1 above)
Comment (vivonk) [reply ui/gulp/build.js]: Should we resolve this conversation @kmehant ?

**Comment by kmehant** [comment]
@vivonk @JaDogg Shall we get this merged?

**Comment by JaDogg** [comment]
@kmehant  I think @singhrohan62 needs to approve
